### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/2fd9fc275b4e571963e5fa1f12547a9097a9b382

### CODE REVIEW

File: configs/pontoon.toml [hunk 1]
```diff
@@ -327,3 +327,9 @@ locales = [
 [[paths]]
     reference = "en/products/vpn/platform-post-download.ftl"
     l10n = "{locale}/products/vpn/platform-post-download.ftl"
+[[paths]]
```
Comment (peiying2) [inline configs/pontoon.toml:330]: Let's move this entry under the en/mozorg/... section, so they are in alphabetical order.

**Review by peiying2 — CHANGES_REQUESTED** [review]
Make sure the file entries are listed in alphabetical order. 

The rest of the PR looks good.

**Review by peiying2 — APPROVED** [review]
Approve this PR. Will bring up the issue in the .toml file separately so it doesn't affect translation.

File: configs/pontoon.toml [hunk 2]
```diff
@@ -327,3 +327,9 @@ locales = [
...
+    reference = "en/mozorg/try-picture-in-picture.ftl.ftl"
```
Comment (flodolo) [inline configs/pontoon.toml:331]: Notice this wasn't showing up in Pontoon, just realized there is a double `.ftl` at the end.

Comment (peiying2) [reply configs/pontoon.toml:331]: I will file an issue on this - more issue than I realized, :).

**Pull request merged**